src/docs/product/data-management-settings/dynamic-sampling/operations.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Dynamic Sampling Operations
+sidebar_order: 30
+description: "Learn more about how dynamic sampling rules are applied."
+---
+
+Dynamic sampling operates on both error events and transaction events.
```

Comment:
I think it would be helpful here to have just two or three sentences about setting up a rule with a link to something more robust. As it is, we've gone right into a discussion of how rules are applied without ever really telling the user how to set one up or even that they need do.